client/src/UIContext.js

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, useState } from 'react';
```

Comment:
Probably makes sense to move this into `/ER`, since it's only used there.  Maybe `ERContext.js`?